.github/workflows/timezone-update.yml

```diff
@@ -0,0 +1,39 @@
+name: Timezone update
+on:
+  schedule:
+    # Run once a week at 00:05 AM UTC on Sunday.
+    - cron: 5 0 * * 0
+
+  workflow_dispatch:
+
+jobs:
+  timezone_update:
+    if: github.repository == 'nodejs/node'
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout nodejs/node
+        uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+
+      - name: Checkout unicode-org/icu-data
+        uses: actions/checkout@v3
+        with:
+          path: icu-data
+          persist-credentials: false
+          repository: unicode-org/icu-data
+
+      - run: ./tools/update-timezone.mjs
+
+      - name: Open Pull Request
+        uses: gr2m/create-or-update-pull-request-action@6720400cad8e74d7adc64640e4e6ea6748b83d8f  # Create a PR or update the Action's existing PR
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_USER_TOKEN }}
+        with:
+          author: Node.js GitHub Bot <github-bot@iojs.org>
+          body: Updates the ICU files.
```

Comment:
should link to documentation about what this is , how to review, what next steps are etc.

should set a team as reviewer such as @nodejs/i18n-api